<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordinateConverter</class>
 <widget class="QDockWidget" name="CoordinateConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coordinate Conversion</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>370</width>
         <height>459</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="xyButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="popupMode">
             <enum>QToolButton::MenuButtonPopup</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="clearFormButton">
            <property name="toolTip">
             <string>Clear form</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="optionsButton">
            <property name="toolTip">
             <string>Settings</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoomButton">
            <property name="toolTip">
             <string>Zoom to coordinate</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="coordCaptureButton">
            <property name="toolTip">
             <string>Capture coordinates by clicking on map</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="wgs84Label">
            <property name="text">
             <string>WGS 84 (lat,lon)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="wgs84LineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="wgs84CopyButton">
              <property name="toolTip">
               <string>Copy to clipboard</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="projectLabel">
            <property name="text">
             <string>- (lat,lon)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="projLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="projCopyButton">
              <property name="toolTip">
               <string>Copy to clipboard</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Custom CRS</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QgsProjectionSelectionWidget" name="customProjectionSelectionWidget"/>
          </item>
          <item row="7" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="customLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="customCopyButton">
              <property name="toolTip">
               <string>Copy to clipboard</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="customLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>- (lat,lon)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Project CRS</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="projectCRSLabel">
            <property name="text">
             <string>EPSG:4326</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dmsLabel">
            <property name="text">
             <string>- D M S.ss</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="dmsLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="dmsCopyButton">
              <property name="toolTip">
               <string>Copy to clipboard</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="dmLabel">
            <property name="text">
             <string>- D M.mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="dmLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="dmCopyButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="ddmmssLabel">
            <property name="text">
             <string>- DDMMSS</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="ddmmssLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="ddmmssCopyButton">
              <property name="toolTip">
               <string>Copy to clipboard</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="utmLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="utmCopyButton">
              <property name="toolTip">
               <string>Copy to clipboard</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Standard UTM</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="customDdmmssLabel">
            <property name="text">
             <string>- DDMMSS</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="customDdmmssLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="customDdmmssCopyButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>UPS</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="upsLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="upsCopyButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>MGRS</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="mgrsLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="mgrsCopyButton">
              <property name="toolTip">
               <string>Copy to clipboard</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Plus Code</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="plusLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="plusCopyButton">
              <property name="toolTip">
               <string>Copy to clipboard</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Geohash</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="geohashLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="geohashCopyButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Maidenhead Grid</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="maidenheadLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="maidenheadCopyButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>GEOREF</string>
            </property>
           </widget>
          </item>
          <item row="15" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="georefLineEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="georefCopyButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
